diagnostics/ENSO_MSE/MSE/moist_routine_hadv.py

```diff
@@ -0,0 +1,122 @@
+import numpy as np

```

Comment:
all of the moist_routine_(variable).py modules can be combined into single routine since you are applying the same calculations to different quantities--just define a general input variable argument. Define a list or dict of variable names to loop through, and call the moist advection routine on the each one.